orion-commons/src/main/java/com/pinterest/orion/utils/TeletraanClient.java

```diff
@@ -0,0 +1,280 @@
+package com.pinterest.orion.utils;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.util.EntityUtils;
+
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.util.logging.Logger;
+
+public class TeletraanClient {
```

Comment:
Please add javadoc for the class (also elsewhere if applicable).

Reply:
Done